$OpenBSD: patch-src_daemon_main_c,v 1.6 2013/05/22 10:31:33 ajacoutot Exp $

XXX deadlock?
(when this is fixed, check that x11/gnome/gdm still needs to install
/var/db/gdm/.pulse)

--- src/daemon/main.c.orig	Thu Jul 19 13:29:38 2012
+++ src/daemon/main.c	Fri Nov 30 11:50:26 2012
@@ -735,6 +735,7 @@ int main(int argc, char *argv[]) {
          * first take the autospawn lock to make things
          * synchronous. */
 
+#if 0
         if ((autospawn_fd = pa_autospawn_lock_init()) < 0) {
             pa_log("Failed to initialize autospawn lock");
             goto finish;
@@ -746,6 +747,7 @@ int main(int argc, char *argv[]) {
         }
 
         autospawn_locked = TRUE;
+#endif
     }
 
     if (conf->daemonize) {
